### PR TITLE
upgrade getting started

### PR DESCRIPTION
**Pull request by karesti**

### CODE REVIEW

File: _layouts/get-started.html [hunk 1]
```diff
@@ -12,12 +12,14 @@ <h2 class="step">Step <span class="step-no">1</span></h2>
     </div>
     <div class="width-10-12 width-12-12-m">
       <h3>Get Infinispan</h3>
-      <p>Add a username and password, then run the one of the following commands:
+      <p>Important: Authentication and Authorization are enabled by default. Use `admin` user name
```
Comment (oraNod) [inline _layouts/get-started.html:15]:
```suggestion
      <p>Infinispan requires authentication and authorization by default. Create a user named `admin`
```

File: _layouts/get-started.html [hunk 2]
```diff
@@ -12,12 +12,14 @@ <h2 class="step">Step <span class="step-no">1</span></h2>
...
+      to implicitly have a user with the admin role.</p>
```
Comment (oraNod) [inline _layouts/get-started.html:16]:
```suggestion
      to get administrative access to Infinispan Server.</p>
```

File: _layouts/get-started.html [hunk 3]
```diff
@@ -12,12 +12,14 @@ <h2 class="step">Step <span class="step-no">1</span></h2>
...
+      <p>Run the one of the following commands:
         <pre><code>
-podman run -it -p 11222:11222 -e USER="username" -e PASS="password" --net=host quay.io/infinispan/server:{{site.data.ispn.stable.minor_version}}
+podman run -it -p 11222:11222 -e USER="admin" -e PASS="pass" --net=host quay.io/infinispan/server:{{site.data.ispn.stable.minor_version}}
```
Comment (oraNod) [inline _layouts/get-started.html:19]: Is there a reason for changing password to pass? IMO "password" is a little bit more meaningful.

**Comment by karesti** [comment]
@oraNod updated

**Review by oraNod — APPROVED** [review]

**Pull request merged**